### ISSUE TITLE
Viash 0.6.5 registered error message on stdout instead of stderr

### ISSUE THREAD

**Issue by DriesSchaumont** [viash-io/viash#312]
Commit ID of the code base that can be used to reproduce the error: [c6d2b4fe399e5becca6ff79364d759447e3e4dce](https://github.com/openpipelines-bio/openpipeline/commit/c6d2b4fe399e5becca6ff79364d759447e3e4dce)
 
When a config cannot be parsed, viash ns list puts the error message on stderr instead of stdout:
Notice the `Not all configs parsed successfully` present in the file.
 ```bash
 ❯ ./bin/viash --version
viash 0.6.5 (c) 2020 Data Intuitive
❯ bin/viash ns list -s src -p nextflow --format json 2> /dev/null > /tmp/ns_list_src.json
❯ inputs=$(jq -r '[.[] | .info.config] | join(";")' /tmp/ns_list_src.json)
parse error: Invalid numeric literal at line 16367, column 2
❯ awk -v nr=10 'FNR>=nr-5 && FNR<=nr+5{ print FNR":"$0 }' /tmp/ns_list_src.json
16362:          "dest" : "par"
16363:        },
16364:        {
16365:          "type" : "file",
16366:          "name" : "--output",
16367:Not all configs parsed successfully
16368:          "alternatives" : [
16369:            "-o"
16370:          ],
16371:          "description" : "Output muon file.",
16372:          "example" : [
 ```

**Comment by DriesSchaumont** [comment]
I also found:

```
        "multiple_sep" : ":",
          "dest" : "par"
  ^[[33m3/57 configs were disabled^[[0m
        }
      ],
      "argument_groups" : [
      ],
      "resources" : [
        {
          "type" : "python_script",
          "path" : "script.py",
          "is_executable" : true,
          "parent" : "file:/Users/driesschaumont/code/openpipeline/src/convert/from_h5ad_to_h5mu/config.vsh.yaml"
        }
  ^[[32m54/57 configs parsed successfully^[[0m
```
Meaning that `viash ns list` also does not work with configs that do have a correct syntax.

**Comment by rcannood** [comment]
Solved in Viash 0.6.6!
